contracts/Comet.sol

```diff
@@ -1121,7 +1121,6 @@ contract Comet is CometMainInterface {
         LiquidatorPoints memory points = liquidatorPoints[absorber];
         points.numAbsorbs++;
         points.numAbsorbed += safe64(accounts.length);
-        points.approxSpend += safe128(gasUsed * block.basefee);
```

Comment:
Curious if you are planning on addressing L-03 as well?

Reply:
I'm not sure why this diff is included here actually, the line has already been deleted in this repo.

I decided not to address L-03, I think it's better to make as few changes to the original Comet contract as possible